examples/docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker Metrics
```

Comment:
What does it look like when you run instrument server on docker along with the docker script. Presumably one is pulling an incomprehensible hostname and one is using a human word set for the container name?